index.html

```diff
@@ -0,0 +1,726 @@
+            </div>
+            <div class="navbar-nav">
+                <button class="navbar-toggler px-1" type="button" data-toggle="collapse" data-target="#navbarNav">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            </div>
+            <div class="collapse navbar-collapse w-100" id="navbarNav">
+                <ul class="navbar-nav align-items-stretch bg-white flex-grow-1  border-bottom">
+                    <li class="nav-item order-1 flex-lg-grow-1">
+                        <a class="nav-link text-body text-right text-lg-center font-weight-bold border-right" href="#">India</a>
+                    </li>
+                    <li class="nav-item order-1 flex-lg-grow-1">
+                        <a class="nav-link text-body text-right text-lg-center font-weight-bold border-right" href="#">World</a>
+                    </li>
+                    <li class="nav-item order-1 flex-lg-grow-1">
+                        <a class="nav-link text-body text-right text-lg-center font-weight-bold border-right" href="#">Business</a>
+                    </li>
+                    <li class="nav-item order-1 flex-lg-grow-1">
+                        <a class="nav-link text-body text-right text-lg-center font-weight-bold border-right" href="#">Tech & Science</a>
+                    </li>
+                    <li class="nav-item order-1 flex-lg-grow-1">
+                        <a class="nav-link text-body text-right text-lg-center font-weight-bold border-right" href="#">Culture</a>
+                    </li>
+                    <li class="nav-item order-1 flex-lg-grow-1">
+                        <a class="nav-link text-body text-right text-lg-center font-weight-bold border-right" href="#">Sports</a>
+                    </li>
+                    <li class="nav-item order-1 flex-lg-grow-1">
+                        <a class="nav-link text-body text-right text-lg-center font-weight-bold border-right" href="#">Health</a>
+                    </li>
+                    <li class="nav-item order-1 flex-lg-grow-1">
+                        <a class="nav-link text-body text-right text-lg-center font-weight-bold border-right" href="#">Opinion</a>
+                    </li>
+                    <li class="nav-item order-1 flex-lg-grow-1">
+                        <a class="nav-link text-body text-right text-lg-center font-weight-bold border-right" href="#">Ndelible</a>
+                    </li>
+                    <li class="nav-item order-0 order-lg-1 flex-lg-grow-1">
+                        <form class="form-inline" method="GET" action="./search-app">
+                            <input class="ml-auto form-control" type="search" placeholder="&#128269;">
+                        </form>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <nav class="navbar navbar-dark bg-danger fixed-top px-1">
```

Comment:
Set the background of that **nav** to `red`

Reply:
ok. I have made the change. But there is no difference between those 2. On the other hand, since this is a bootstrap project, there should be less emphasis of specifying these details through the CSS. The point should be to write as less CSS as possible, and making use of the Bootstrap classes.